### DISCUSSION TITLE
Deactivate Enter purpose of visit

### DISCUSSION THREAD

**Discussion by mediakanal** [YetiForceCompany/YetiForceCRM#14497] [How to] [answered]
Hello, as an admin, I am always asked to enter the purpose of the visit after logging in. How do I deactivate this query?

regards
Simon Aquilini

**Comment by rskrzypczak** [comment, marked as answer]
stable: https://github.com/YetiForceCompany/YetiForceCRM/issues/13883
dev: https://user-images.githubusercontent.com/25615135/106721770-60cf4c80-6605-11eb-8316-6248f10e0c6d.gif

**Reply by mediakanal** [reply]
I don't see this selection for me
![Bildschirmfoto 2021-04-02 um 17 31 52](https://user-images.githubusercontent.com/79196539/113430023-b8a5ec00-93d9-11eb-806f-30d564fae7ff.png)

**Reply by rskrzypczak** [reply]
You can't see because you don't have a development version.
You need to change the parameter in the file (stable: https://github.com/YetiForceCompany/YetiForceCRM/issues/13883):
https://github.com/YetiForceCompany/YetiForceCRM/blob/20f376f9c8ea5866f80e7eb6e77c4fc246bcb10e/config/Security.php#L197-L198

**Reply by mediakanal** [reply]
Perfect thanks for the very good and fast support.
